package.json

```diff
@@ -48,9 +48,11 @@
     "test": "lerna run --stream test",
     "test.watch": "lerna run --stream test.watch",
     "ci-build-no-rust": "lerna run --ignore @temporalio/core-bridge --stream build",
+    "ci-per-change-load": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url='' --max-cached-wfs 500",
```

Comment:
Wait, are we using this now? It's included in `ci-stress`

Reply:
That's fine, the idea is to have something short we can run per-pipeline and the nightly runs run stress, which maybe also includes this